src/OpenTelemetry.Contrib.Exporter.Stackdriver/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## Unreleased
+## 1.1.0-beta1

```

Comment:
would be nice to use 1.0.0-beta1 :)

Reply:
Oh yeah duh. Just because it supports that version of the main package doesn't mean it can't be 1.0.0. I'm short of sleep lol.